vue-components/src/components/Button.vue

```diff
@@ -1,5 +1,12 @@
 <template>
-	<button :class="[ 'wikit', 'wikit-Button', `wikit-Button--${ type }` ]">
+	<button
+		:class="[
+			'wikit',
+			'wikit-Button',
+			(quiet === true) ? `wikit-Button--quiet` : '',
+			(quiet === false) ? `wikit-Button--${type}` : `wikit-Button--quiet--${type}`
```

Comment:
Just fyi, we'll end up having 3 main button classes/variants:

- --quiet (neutral, progressive, destructive)
- --primary (progressive, destructive)
- --normal (aka framed: we currently have a neutral flavor, but it's possible that progressive and destructive types are included in the future)